generators/server/templates/src/main/java/_package_/security/JHipsterOpenIdUserDetailsMapper.java.ejs

```diff
@@ -8,6 +8,7 @@ import com.nimbusds.jwt.JWT;
 import com.nimbusds.jwt.JWTClaimsSet;
 import com.nimbusds.jwt.JWTParser;
```

Comment:
It looks like the following dependency needs to be added for this class to compile:

```xml
<dependency>
    <groupId>com.nimbusds</groupId>
    <artifactId>nimbus-jose-jwt</artifactId>
</dependency>
```